### ISSUE TITLE
iOS app crashes on startup

### ISSUE THREAD

**Issue by afaden** [glushchenko/fsnotes#345]
When I launch FSNotes for iOS v1.7 on my iPhone or iPad running iOS 11.4.1, it shows a white screen for a while and then crashes back to the home screen. From the console log it looks like it's trying to load all of my notes and running out of time. I see a whole lot of these messages:
```
default 11:27:43.765962 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d025a0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted] documentState: [Closed]
```
And eventually I see this:
```
default	11:28:03.631985 -0500	SpringBoard	Process exited: <FBApplicationProcess: 0x10ad19940; FSNotes iOS (co.fluder.mobile.FSNotes-iOS); pid: -1> -> <FBApplicationProcessExitContext: 0x1c044a9b0; exitReason: (none); terminationReason: (none)> {
 stateAtExit = <FBProcessState: 0x1c1026960; pid: 5932; taskState: Running; visibility: Foreground>;
 terminationRequest = <FBSProcessTerminationRequest: 0x1c28714c0; label: "watchdog provision violated"; exceptionCode: "Watchdog Violation (0x8BADF00D)"; reportType: CrashLog; explanation: "scene-create watchdog transgression: co.fluder.mobile.FSNotes-iOS exhausted real (wall clock) time allowance of 19.89 seconds">;
 watchdogContext = <FBProcessWatchdogEventContext: 0x1c205cd70; event: scene-create> {
 updateContext = <FBSceneUpdateContext: 0x1c2e6bd80; scene: co.fluder.mobile.FSNotes-iOS; tID: 1> {
 settings = <UIApplicationSceneSettings: 0x1c0cc2fb0> {
 displayConfiguration = <FBSDisplayConfiguration: 0x1c4164f80; Main; mode: "414x736@3x 60Hz p3 SDR"> {
```

Here's the full log:
```
default	11:27:42.954024 -0500	SpringBoard	Running <SBAppToAppWorkspaceTransaction: 0x11c1fa440> for transition request:
<SBMainWorkspaceTransitionRequest: 0x1c50fef00; eventLabel: SBUIApplicationIconLaunchEventLabel; display: Main; source: HomeScreen> {
 applicationContext = <SBWorkspaceApplicationSceneTransitionContext: 0x1c4154060; background: NO> entities = {
 SBLayoutRolePrimary = <SBDeviceApplicationSceneEntity: 0x1c0482620; ID: co.fluder.mobile.FSNotes-iOS; layoutRole: primary>;
 };
}
default	11:27:43.004261 -0500	assertiond	Submitting new job for "co.fluder.mobile.FSNotes-iOS" on behalf of <BKProcess: 0x100a5f870; SpringBoard; com.apple.springboard; pid: 5360; agency: SystemShell; visibility: foreground; task: running>
default	11:27:43.004320 -0500	SpringBoard	Bootstrapping co.fluder.mobile.FSNotes-iOS with intent foreground-interactive
default	11:27:43.004569 -0500	assertiond	Submitted job with label: UIKitApplication:co.fluder.mobile.FSNotes-iOS[0x6d1][73]
default	11:27:43.017757 -0500	assertiond	[FSNotes iOS:5932] Adding client: <BKProcessInfoServerClient: 0x10096e870; pid: 5360>
default	11:27:43.017842 -0500	assertiond	[FSNotes iOS:5932] Setting jetsam priority to 10 [0x10000]
default	11:27:43.017996 -0500	assertiond	Now tracking process <BKProcess: 0x100987b60; FSNotes iOS; co.fluder.mobile.FSNotes-iOS; pid: 5932; agency: Application; visibility: foreground; task: running; hostpid: 5360> with host <BKProcess: 0x100a5f870; SpringBoard; com.apple.springboard; pid: 5360; agency: SystemShell; visibility: foreground; task: running>
default	11:27:43.018203 -0500	symptomsd	5932 co.fluder.mobile.FSNotes-iOS: ForegroundRunning (most elevated: ForegroundRunning)
default	11:27:43.018782 -0500	mediaserverd	-CMSessionMgr- CMSessionMgrHandleApplicationStateChange: CMSession: Client co.fluder.mobile.FSNotes-iOS with pid '5932' is now Foreground Running. Background entitlement: NO
default	11:27:43.024329 -0500	SpringBoard	WIFI PICKER [co.fluder.mobile.FSNotes-iOS]: isProcessLaunch: 1, isForegroundActivation: 1, isForegroundDeactivation: 0
default	11:27:43.024410 -0500	SpringBoard	[co.fluder.mobile.FSNotes-iOS] Bootstrap complete with label: UIKitApplication:co.fluder.mobile.FSNotes-iOS[0x6d1][73]
default	11:27:43.025010 -0500	SpringBoard	[FBProcessManager] Adding: <FBApplicationProcess: 0x10ad19940; FSNotes iOS (co.fluder.mobile.FSNotes-iOS); pid: 5932>
default	11:27:43.035941 -0500	assertiond	[SpringBoard:5360] Attempting to acquire assertion for FSNotes iOS:5932: <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4>
default	11:27:43.041986 -0500	assertiond	[FSNotes iOS:5932] Add assertion: <BKProcessAssertion: 0x100981630; id: 5360-B20A114F-4AFD-4E01-9BF1-2B5403AA9FD4; name: UIApplicationLaunch; state: active; reason: activation; duration: infs> {
 owner = <BSProcessHandle: 0x10090e4c0; SpringBoard:5360; valid: YES>;
 flags = preventSuspend, preventThrottleDownUI, preventThrottleDownCPU, wantsForegroundResourcePriority, preventSuspendOnSleep;
}
default	11:27:43.042127 -0500	assertiond	[FSNotes iOS:5932] Activate assertion: <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4>
default	11:27:43.042285 -0500	assertiond	[FSNotes iOS:5932] New process assertion state; preventSuspend, preventThrottleDownUI, preventThrottleDownCPU, wantsForegroundResourcePriority, preventSuspendOnSleep (assertion 0x100981630 added: preventSuspend, preventThrottleDownUI, preventThrottleDownCPU, wantsForegroundResourcePriority, preventSuspendOnSleep; removed: (none))
default	11:27:43.042356 -0500	assertiond	[FSNotes iOS:5932] Setting jetsam priority to 10 [0x10100]
default	11:27:43.047445 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:1 (CPUMON check): {
 <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4> [active]
}
default	11:27:43.059909 -0500	assertiond	[FSNotes iOS:5932] setpriority success for resource GPU to PRIO_DARWIN_GPU_ALLOW
default	11:27:43.064599 -0500	assertiond	[FSNotes iOS:5932] setpriority success for resource CPU to default (0)
default	11:27:43.289866 -0500	SpringBoard	[co.fluder.mobile.FSNotes-iOS] Setting deactivation reasons to: 'systemAnimation' for reason: scene settings update - settings are eligible for deactivation reasons.
default	11:27:43.301424 -0500	SpringBoard	Application process state changed for co.fluder.mobile.FSNotes-iOS: <SBApplicationProcessState: 0x1c1a35100; pid: 5932; taskState: Running; visibility: Unknown>
default	11:27:43.308856 -0500	assertiond	[FSNotes iOS:5932] workspaceConnectedWithTaskPortRight: received task port
default	11:27:43.308998 -0500	FSNotes iOS	Retrieving resting unlock: 0
default	11:27:43.314107 -0500	SpringBoard	[FBSystemServiceServer] Client FSNotes iOS:5932 connected to service com.apple.frontboardservices.system-app-proxy
default	11:27:43.321001 -0500	SpringBoard	[co.fluder.mobile.FSNotes-iOS] Sending scene action [Logical Activate] through WorkspaceServer: 0x1c0c9d330
default	11:27:43.322749 -0500	assertiond	[SpringBoard:5360] Attempting to acquire assertion for FSNotes iOS:5932: <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882>
default	11:27:43.322827 -0500	assertiond	[FSNotes iOS:5932] Add assertion: <BKProcessAssertion: 0x100a81b40; id: 5360-F7B77A47-F306-4FC3-9B4B-6D04343A1882; name: Resume; state: active; reason: activation; duration: infs> {
 owner = <BSProcessHandle: 0x10090e4c0; SpringBoard:5360; valid: YES>;
 flags = preventSuspend, preventThrottleDownUI, preventThrottleDownCPU, preventSuspendOnSleep;
}
default	11:27:43.322880 -0500	assertiond	[FSNotes iOS:5932] Activate assertion: <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882>
default	11:27:43.322916 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:2 (CPUMON check): {
 <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4> [active]
 <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882> [active]
}
default	11:27:43.340369 -0500	assertiond	[SpringBoard:5360] Attempting to acquire assertion for FSNotes iOS:5932: <BKProcessAssertion: 0x100a6afb0; "Deliver Message" (suspend:10s); id:…FE00B4616552>
default	11:27:43.340541 -0500	assertiond	[FSNotes iOS:5932] Add assertion: <BKProcessAssertion: 0x100a6afb0; id: 5360-6288F728-E2B2-4195-A7DB-FE00B4616552; name: "Deliver Message"; state: active; reason: suspend; duration: 10.0s> {
 owner = <BSProcessHandle: 0x10090e4c0; SpringBoard:5360; valid: YES>;
 flags = preventSuspend, preventThrottleDownUI, preventThrottleDownCPU, preventSuspendOnSleep;
}
default	11:27:43.340887 -0500	assertiond	[FSNotes iOS:5932] Activate assertion: <BKProcessAssertion: 0x100a6afb0; "Deliver Message" (suspend:10s); id:…FE00B4616552>
default	11:27:43.341034 -0500	assertiond	[FSNotes iOS:5932] Setting jetsam priority to 10 [0x10300]
default	11:27:43.341149 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:3 (CPUMON check): {
 <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4> [active]
 <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882> [active]
 <BKProcessAssertion: 0x100a6afb0; "Deliver Message" (suspend:10s); id:…FE00B4616552> [active]
}
default	11:27:43.342878 -0500	SpringBoard	[co.fluder.mobile.FSNotes-iOS] Sending scene action [SceneLifecycleEventOnly] through WorkspaceServer: 0x1c0c9d330
default	11:27:43.343526 -0500	assertiond	[FSNotes iOS:5932] Deactivate assertion: <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4>
default	11:27:43.346246 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:3 (deactivateAssertion): {
 <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882> [active]
 <BKProcessAssertion: 0x100a6afb0; "Deliver Message" (suspend:10s); id:…FE00B4616552> [active]
}
default	11:27:43.346725 -0500	assertiond	[FSNotes iOS:5932] Scheduling allow-idle-sleep timer with interval: 180.0
default	11:27:43.346817 -0500	assertiond	[FSNotes iOS:5932] New process assertion state; preventSuspend, preventThrottleDownUI, preventThrottleDownCPU, preventSuspendOnSleep (assertion 0x100981630 added: (none); removed: wantsForegroundResourcePriority)
default	11:27:43.346875 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:3 (CPUMON check): {
 <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882> [active]
 <BKProcessAssertion: 0x100a6afb0; "Deliver Message" (suspend:10s); id:…FE00B4616552> [active]
}
default	11:27:43.347085 -0500	assertiond	[FSNotes iOS:5932] Remove assertion: <BKProcessAssertion: 0x100981630; "UIApplicationLaunch" (activation:inf); id:…2B5403AA9FD4>
default	11:27:43.347182 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:3 (CPUMON check): {
 <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882> [active]
 <BKProcessAssertion: 0x100a6afb0; "Deliver Message" (suspend:10s); id:…FE00B4616552> [active]
}
default	11:27:43.368652 -0500	SpringBoard	Application process state changed for co.fluder.mobile.FSNotes-iOS: <SBApplicationProcessState: 0x1c4e31900; pid: 5932; taskState: Running; visibility: Foreground>
default	11:27:43.369249 -0500	SpringBoard	Application process state changed for co.fluder.mobile.FSNotes-iOS: <SBApplicationProcessState: 0x1c4037740; pid: 5932; taskState: Running; visibility: Foreground>
default	11:27:43.509410 -0500	SpringBoard	[co.fluder.mobile.FSNotes-iOS] Setting deactivation reasons to: '(none)' for reason: updateAllScenesForBand - Assertion removed.
default	11:27:43.510005 -0500	SpringBoard	[co.fluder.mobile.FSNotes-iOS] Sending scene action [SceneLifecycleEventOnly] through WorkspaceServer: 0x1c0c9d330
default	11:27:43.512330 -0500	SpringBoard	[FBSystemServiceServer] Client FSNotes iOS:5932 connected to service com.apple.springboardservices.application
error	11:27:43.666081 -0500	FSNotes iOS	CUICatalog: Invalid asset name supplied: ''
default	11:27:43.666250 -0500	FSNotes iOS	Could not load the "" image referenced from a nib in the bundle with identifier "co.fluder.mobile.FSNotes-iOS"
default	11:27:43.719163 -0500	symptomsd	Entry, display name co.fluder.mobile.FSNotes-iOS uuid 105528E1-FDC1-39C0-A1B8-B10882036641 pid 5932 isFront 1
default	11:27:43.719436 -0500	symptomsd	Continue with bundle name co.fluder.mobile.FSNotes-iOS, is front 1
default	11:27:43.720028 -0500	symptomsd	co.fluder.mobile.FSNotes-iOS: Foreground: true
default	11:27:43.752703 -0500	symptomsd	Failed to find process for co.fluder.mobile.FSNotes-iOS
default 11:27:43.765962 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d025a0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 1] documentState: [Closed]
default 11:27:43.928074 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d01ef0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 2] documentState: [Closed]
default 11:27:44.198325 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113ef5600> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 3] documentState: [Closed]
default 11:27:44.236243 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d59df0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 4] documentState: [Closed]
default 11:27:44.406881 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113ef7870> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 5] documentState: [Closed]
default 11:27:44.409382 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d5b170> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 6] documentState: [Closed]
default 11:27:44.447746 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113ef8c20> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 7] documentState: [Closed]
default 11:27:44.452277 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d5ce20> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 8] documentState: [Closed]
default 11:27:44.481255 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113ef90b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 9] documentState: [Closed]
default 11:27:44.730195 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d5d5a0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 10] documentState: [Closed]
default 11:27:44.793470 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d5e260> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 11] documentState: [Closed]
default 11:27:44.962498 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d5ec10> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 12] documentState: [Closed]
default 11:27:45.287477 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11594acb0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 13] documentState: [Closed]
default 11:27:45.293590 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d622f0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 14] documentState: [Closed]
default 11:27:45.331028 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d62450> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 15] documentState: [Closed]
default 11:27:45.473419 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11594c130> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 16] documentState: [Closed]
default 11:27:45.507847 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d630d0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 17] documentState: [Closed]
default 11:27:45.561436 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d63700> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 18] documentState: [Closed]
default 11:27:46.361201 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d66700> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 19] documentState: [Closed]
default 11:27:46.590465 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d66df0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 20] documentState: [Closed]
default 11:27:46.703242 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115954300> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 21] documentState: [Closed]
default 11:27:46.784551 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115955880> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 22] documentState: [Closed]
default 11:27:46.943148 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159559e0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 23] documentState: [Closed]
default 11:27:46.993939 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d69590> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 24] documentState: [Closed]
default 11:27:47.011481 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159560d0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 25] documentState: [Closed]
default 11:27:47.068928 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115956bd0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 26] documentState: [Closed]
default 11:27:47.244873 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d6a720> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 27] documentState: [Closed]
default 11:27:47.978063 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d6de40> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 28] documentState: [Closed]
default 11:27:48.112383 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11595ac10> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 29] documentState: [Closed]
default 11:27:48.262052 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d6fd60> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 30] documentState: [Closed]
default 11:27:48.305059 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11595dae0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 31] documentState: [Closed]
default 11:27:48.668200 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d729e0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 32] documentState: [Closed]
default 11:27:48.822733 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d72cb0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 33] documentState: [Closed]
default 11:27:48.849265 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11595fd40> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 34] documentState: [Closed]
default 11:27:49.083477 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115961ba0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 35] documentState: [Closed]
default 11:27:49.289002 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d75f00> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 36] documentState: [Closed]
default 11:27:49.443678 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115963a90> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 37] documentState: [Closed]
default 11:27:49.514885 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115963f70> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 38] documentState: [Closed]
default 11:27:49.677780 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115965ef0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 39] documentState: [Closed]
default 11:27:49.744081 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d77280> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 40] documentState: [Closed]
default 11:27:50.012391 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d798a0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 41] documentState: [Closed]
default 11:27:50.031661 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115968690> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 42] documentState: [Closed]
default 11:27:50.060185 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115969300> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 43] documentState: [Closed]
default 11:27:50.134038 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115969630> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 44] documentState: [Closed]
default 11:27:50.416176 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11596ab20> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 45] documentState: [Closed]
default 11:27:50.424495 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11596b390> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 46] documentState: [Closed]
default 11:27:50.445863 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11596ba80> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 47] documentState: [Closed]
default 11:27:50.484994 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11596bf60> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 48] documentState: [Closed]
default 11:27:50.495676 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d7ba70> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 49] documentState: [Closed]
default 11:27:50.796598 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d7bd40> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 50] documentState: [Closed]
default 11:27:50.886015 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d7c220> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 51] documentState: [Closed]
default 11:27:51.197049 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d7ec10> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 52] documentState: [Closed]
default 11:27:51.240297 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d7f0f0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 53] documentState: [Closed]
default 11:27:51.358122 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d7fcb0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 54] documentState: [Closed]
default 11:27:51.369105 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11596fdf0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 55] documentState: [Closed]
default 11:27:51.434894 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115970c80> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 56] documentState: [Closed]
default 11:27:51.688726 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115971df0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 57] documentState: [Closed]
default 11:27:51.737725 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d810a0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 58] documentState: [Closed]
default 11:27:51.820332 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d81b70> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 59] documentState: [Closed]
default 11:27:51.838877 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d83870> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 60] documentState: [Closed]
default 11:27:52.057507 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d84430> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 61] documentState: [Closed]
default 11:27:52.088085 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d84be0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 62] documentState: [Closed]
default 11:27:52.091651 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d85020> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 63] documentState: [Closed]
default 11:27:52.311918 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115974f20> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 64] documentState: [Closed]
default 11:27:52.353189 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d85690> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 65] documentState: [Closed]
default 11:27:52.393122 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115975610> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 66] documentState: [Closed]
default 11:27:52.401843 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d85d90> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 67] documentState: [Closed]
default 11:27:52.850285 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159777b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 68] documentState: [Closed]
default 11:27:52.902901 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115977ea0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 69] documentState: [Closed]
default 11:27:53.061173 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11597a340> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 70] documentState: [Closed]
default 11:27:53.082429 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11597a610> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 71] documentState: [Closed]
default 11:27:53.279417 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d8c1e0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 72] documentState: [Closed]
default 11:27:53.438265 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d8c4b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 73] documentState: [Closed]
default 11:27:53.458782 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d8c990> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 74] documentState: [Closed]
default 11:27:53.723740 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d8cd30> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 75] documentState: [Closed]
default 11:27:53.856559 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11597cfb0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 76] documentState: [Closed]
default 11:27:54.023892 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11597ded0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 77] documentState: [Closed]
default 11:27:54.064257 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11597d940> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 78] documentState: [Closed]
default 11:27:54.333500 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d8eb00> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 79] documentState: [Closed]
default 11:27:54.437755 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115980d60> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 80] documentState: [Closed]
default 11:27:54.452930 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159812b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 81] documentState: [Closed]
default 11:27:54.467909 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115981cf0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 82] documentState: [Closed]
default 11:27:54.522704 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d903b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 83] documentState: [Closed]
default 11:27:54.647832 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115982e20> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 84] documentState: [Closed]
default 11:27:54.835557 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159857d0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 85] documentState: [Closed]
default 11:27:54.962308 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115985aa0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 86] documentState: [Closed]
default 11:27:55.090592 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115985d70> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 87] documentState: [Closed]
default 11:27:55.118511 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115986250> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 88] documentState: [Closed]
default 11:27:55.471943 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115987300> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 89] documentState: [Closed]
default 11:27:55.653044 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d94a50> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 90] documentState: [Closed]
default 11:27:55.713714 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d95210> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 91] documentState: [Closed]
default 11:27:55.720562 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115989a40> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 92] documentState: [Closed]
default 11:27:55.963537 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d954e0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 93] documentState: [Closed]
default 11:27:56.013361 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115989fe0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 94] documentState: [Closed]
default 11:27:56.033067 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11598a9d0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 95] documentState: [Closed]
default 11:27:56.639587 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da3760> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 96] documentState: [Closed]
default 11:27:56.682020 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11598e130> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 97] documentState: [Closed]
default 11:27:56.955970 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da46c0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 98] documentState: [Closed]
default 11:27:57.010253 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11598f880> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 99] documentState: [Closed]
default 11:27:57.231767 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159914b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 100] documentState: [Closed]
default 11:27:57.396287 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115991990> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 101] documentState: [Closed]
default 11:27:57.406480 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da6430> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 102] documentState: [Closed]
default 11:27:57.416970 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da6910> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 103] documentState: [Closed]
default 11:27:57.505440 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115991f30> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 104] documentState: [Closed]
default 11:27:57.526960 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da7030> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 105] documentState: [Closed]
default 11:27:57.577704 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115992090> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 106] documentState: [Closed]
default 11:27:57.733375 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115992d50> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 107] documentState: [Closed]
default 11:27:57.858393 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da9330> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 108] documentState: [Closed]
default 11:27:57.951201 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113da9b30> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 109] documentState: [Closed]
default 11:27:58.022682 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113daa7b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 110] documentState: [Closed]
default 11:27:58.164508 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x115996290> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 111] documentState: [Closed]
default 11:27:58.223884 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113daa4f0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 112] documentState: [Closed]
default 11:27:58.781630 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159988a0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 113] documentState: [Closed]
default 11:27:59.012823 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113dade20> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 114] documentState: [Closed]
default 11:27:59.015617 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11599bbb0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 115] documentState: [Closed]
default 11:27:59.106348 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11599cba0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 116] documentState: [Closed]
default 11:27:59.332151 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11599ce70> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 117] documentState: [Closed]
default 11:27:59.610048 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x11599d290> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 118] documentState: [Closed]
default 11:28:00.198955 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159a13c0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 119] documentState: [Closed]
default 11:28:01.326173 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159a5170> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 120] documentState: [Closed]
default 11:28:01.394958 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159a5cf0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 121] documentState: [Closed]
default 11:28:01.688999 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159a74c0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 122] documentState: [Closed]
default 11:28:01.788569 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113dba540> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 123] documentState: [Closed]
default 11:28:01.856273 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159a8130> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 124] documentState: [Closed]
default 11:28:01.977680 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159a8400> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 125] documentState: [Closed]
default 11:28:02.668283 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159ab750> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 126] documentState: [Closed]
default 11:28:02.686650 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159abf00> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 127] documentState: [Closed]
default 11:28:02.858434 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x113d0e2f0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 128] documentState: [Closed]
default 11:28:02.957449 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159acee0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 129] documentState: [Closed]
default 11:28:02.997379 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159ad1b0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 130] documentState: [Closed]
default 11:28:03.327492 -0500   FSNotes iOS initialized document instance: <FSNotes_iOS.Note: 0x1159af6d0> fileURL: file:///private/var/mobile/Library/Mobile%20Documents/iCloud~co~fluder~fsnotes/Documents/[redacted 131] documentState: [Closed]
default	11:28:03.516455 -0500	assertiond	[FSNotes iOS:5932] Port death watcher fired.
default	11:28:03.517583 -0500	assertiond	Server invalidated <BKProcessAssertion: 0x100a82240; "FBSceneSnapshotAction:co.fluder.mobile.FSNotes-iOS" (finishTask:180s); id:…B8A3C17BFE15>
default	11:28:03.518346 -0500	assertiond	Client relinquished <BKProcessAssertion: 0x100a82240; "FBSceneSnapshotAction:co.fluder.mobile.FSNotes-iOS" (finishTask:180s); id:…B8A3C17BFE15>
default	11:28:03.518908 -0500	assertiond	[FSNotes iOS:5932] Ignoring assertion remove, because we are terminated or pending termination
default	11:28:03.519059 -0500	assertiond	[FSNotes iOS:5932] Ignoring assertion remove, because we are terminated or pending termination
default	11:28:03.519972 -0500	assertiond	Process exited: <BKProcess: 0x100987b60; FSNotes iOS; co.fluder.mobile.FSNotes-iOS; pid: 5932; agency: Application; visibility: foreground; task: none; hostpid: 5360>
default	11:28:03.520030 -0500	assertiond	[FSNotes iOS:5932] Invalidating...
default	11:28:03.520095 -0500	assertiond	Releasing PowerAssertion on FSNotes iOS:5932 from invalidate
default	11:28:03.520154 -0500	assertiond	[FSNotes iOS:5932] dump all assertions HWM:3 (CPUMON check): {
 <BKProcessAssertion: 0x100a81b40; "Resume" (activation:inf); id:…6D04343A1882> [active]
 <BKProcessAssertion: 0x100a82240; "FBSceneSnapshotAction:co.fluder.mobile.FSNotes-iOS" (finishTask:180s); id:…B8A3C17BFE15> [active]
}
default	11:28:03.520196 -0500	powerd	Process assertiond.73 Released SystemIsActive "FSNotes iOS:5932:5360-5E573BCF-E132-4E35-95B6-B8A3C17BFE15 [FBSceneSnapshotAction:co.fluder.mobile.FSNotes-iOS] [0x100a82240]" age:00:00:00  id:51539641733 [System: PrevIdle SysAct]
default	11:28:03.628649 -0500	assertiond	[FSNotes iOS:5932] Got exit context: <BKSProcessExitContext: 0x100a7cfb0; reason: (none)>
default	11:28:03.630559 -0500	SpringBoard	<FBApplicationProcess: 0x10ad19940; FSNotes iOS (co.fluder.mobile.FSNotes-iOS); pid: 5932> exited.
default	11:28:03.630910 -0500	SpringBoard	[FBProcessManager] Removing: <FBApplicationProcess: 0x10ad19940; FSNotes iOS (co.fluder.mobile.FSNotes-iOS); pid: -1>
default	11:28:03.631985 -0500	SpringBoard	Process exited: <FBApplicationProcess: 0x10ad19940; FSNotes iOS (co.fluder.mobile.FSNotes-iOS); pid: -1> -> <FBApplicationProcessExitContext: 0x1c044a9b0; exitReason: (none); terminationReason: (none)> {
 stateAtExit = <FBProcessState: 0x1c1026960; pid: 5932; taskState: Running; visibility: Foreground>;
 terminationRequest = <FBSProcessTerminationRequest: 0x1c28714c0; label: "watchdog provision violated"; exceptionCode: "Watchdog Violation (0x8BADF00D)"; reportType: CrashLog; explanation: "scene-create watchdog transgression: co.fluder.mobile.FSNotes-iOS exhausted real (wall clock) time allowance of 19.89 seconds">;
 watchdogContext = <FBProcessWatchdogEventContext: 0x1c205cd70; event: scene-create> {
 updateContext = <FBSceneUpdateContext: 0x1c2e6bd80; scene: co.fluder.mobile.FSNotes-iOS; tID: 1> {
 settings = <UIApplicationSceneSettings: 0x1c0cc2fb0> {
 displayConfiguration = <FBSDisplayConfiguration: 0x1c4164f80; Main; mode: "414x736@3x 60Hz p3 SDR"> {
default	11:28:03.632646 -0500	SpringBoard	Application process state changed for co.fluder.mobile.FSNotes-iOS: (null)
default	11:28:03.633099 -0500	assertiond	Deleted job with label: UIKitApplication:co.fluder.mobile.FSNotes-iOS[0x6d1][73]
default	11:28:03.633318 -0500	assertiond	[FSNotes iOS:5932] Deleted launchd job with label: UIKitApplication:co.fluder.mobile.FSNotes-iOS[0x6d1][73]
default	11:28:03.633663 -0500	assertiond	[FSNotes iOS:5932] Invalidation complete.
default	11:28:03.644934 -0500	symptomsd	5932 co.fluder.mobile.FSNotes-iOS: Terminated (most elevated: Terminated)
default	11:28:03.645005 -0500	mediaserverd	-CMSessionMgr- CMSessionMgrHandleApplicationStateChange: CMSession: Client co.fluder.mobile.FSNotes-iOS with pid '5932' is now Terminated. Background entitlement: NO
default	11:28:03.645083 -0500	assertiond	Checking for deferred bootstrap request for co.fluder.mobile.FSNotes-iOS
default	11:28:03.645126 -0500	SpringBoard	WIFI PICKER [co.fluder.mobile.FSNotes-iOS]: isProcessLaunch: 0, isForegroundActivation: 0, isForegroundDeactivation: 1
default	11:28:03.645312 -0500	symptomsd	Entry, display name co.fluder.mobile.FSNotes-iOS uuid (null) pid 5932 isFront 0
default	11:28:03.645542 -0500	symptomsd	Continue with bundle name co.fluder.mobile.FSNotes-iOS, is front 0
default	11:28:03.645680 -0500	symptomsd	co.fluder.mobile.FSNotes-iOS: Foreground: false
default	11:28:03.646540 -0500	symptomsd	Failed to find process for co.fluder.mobile.FSNotes-iOS
default	11:28:03.648662 -0500	assertiond	[FSNotes iOS:5932] Removing client: <BKProcessInfoServerClient: 0x10096e870; pid: 5360>
default	11:28:03.648764 -0500	assertiond	[FSNotes iOS:5932] No clients remain.
default	11:28:03.648811 -0500	assertiond	[FSNotes iOS:5932] Terminating because the job-submitter has disconnected.
```

BTW, FSNotes for Mac v2.0.9 is running fine with the same folder of notes.

**Comment by glushchenko** [comment]
Don't know how to reproduce. Tested with 11.4.1 and iOS 12.

**Comment by afaden** [comment]
I moved all of my notes out of my FSNotes folder on iCloud Drive to check that the app would start up, and it did. Then I tried to reproduce the problem from scratch using test files:
```
for ((i=1;i<=1000;i++)); do
    echo "test" > $i.txt
done
```
I added them to my FSNotes folder in batches of 100 to see how many it could handle before it crashed on startup. It started crashing for me once there were 500 test files.

**Comment by afaden** [comment]
Apparently the [watchdog mechanism](https://developer.apple.com/library/archive/qa/qa1693/_index.html) kills the app if the main thread takes too long to reach applicationDidBecomeActive. To debug the problem, I ran the app in the iOS Simulator using a test iCloud Drive with a few thousand text files. I couldn't get it to crash in the Simulator, apparently [the watchdog mechanism only applies to real devices](
https://developer.apple.com/videos/play/wwdc2018-414/?time=1339). But I did see why it's taking so long to reach applicationDidBecomeActive. In ViewController.viewDidLoad, it's currently doing this:
```
storage.initiateCloudDriveSync()
storage.loadDocuments()
updateTable() {}
```
loadDocuments takes a long time, and it's run synchronously in the main thread, blocking super.viewDidLoad from being called, and leading to the watchdog timeout.

One potential solution is to run loadDocuments in another thread. I tried that out based on [this guide](https://www.raywenderlich.com/5370-grand-central-dispatch-tutorial-for-swift-4-part-1-2):
```
storage.initiateCloudDriveSync()
DispatchQueue.global(qos: .userInitiated).async { [weak self] in
    guard let outerself = self else {
        return
    }
    // kick off loading documents
    outerself.storage.loadDocuments()
    DispatchQueue.main.async { [weak outerself] in
        // when that completes, update table
        outerself?.updateTable() {}
    }
}
```
It seems to work, it hits applicationDidBecomeActive almost instantly and the app starts up with an empty table, and then eventually it finishes loadDocuments and populates the table. I think this will prevent the app from crashing on startup.

However, on launch it'll still be a while before you can actually do anything with your notes, so maybe it should show some kind of loading indicator. Or is there any way to speed up loadDocuments? I thought it might be slow because it's downloading all of the files, but I tried it while offline and the Files app still showed everything, so I think they're already stored locally, but it was still slow. I tried profiling the app in Instruments but I'm not sure if it shows a smoking gun (note that the wall time was much longer than it shows):
```
Weight      Self Weight      Symbol Name
------      -----------      -----------
3.81 s  100.0%  0 s          FSNotes iOS (49421)
2.69 s   70.6%  0 s           _dispatch_worker_thread3  0x5d6c23
2.69 s   70.6%  0 s            start_wqthread
2.69 s   70.6%  0 s             _pthread_wqthread
2.69 s   70.6%  0 s              _dispatch_worker_thread3
2.69 s   70.6%  0 s               _dispatch_root_queue_drain
2.69 s   70.6%  0 s                _dispatch_client_callout
2.69 s   70.6%  0 s                 _dispatch_call_block_and_release
2.69 s   70.6%  0 s                  _T0Ieg_IeyB_TR
2.69 s   70.6%  0 s                   partial apply for closure #1 in ViewController.viewDidLoad()
2.69 s   70.6%  0 s                    closure #1 in ViewController.viewDidLoad()
2.69 s   70.6%  0 s                     Storage.loadDocuments(tryCount:)
2.69 s   70.6%  0 s                      Storage.loadLabel(_:shouldScanCache:)
2.69 s   70.5%  2.00 ms                   specialized Storage.loadLabel(_:shouldScanCache:)
1.01 s   26.4%  0 s                        Note.load(_:)
1.01 s   26.4%  0 s                         specialized Note.load(_:)
703.00 ms   18.4%   0 s                      Note.getContent()
688.00 ms   18.0%   1.00 ms                   specialized @nonobjc NSAttributedString.init(url:options:documentAttributes:)
680.00 ms   17.8%   1.00 ms                    -[NSAttributedString(NSAttributedStringUIFoundationAdditions) initWithURL:options:documentAttributes:error:]
642.00 ms   16.8%   6.00 ms                     _NSReadAttributedStringFromURLOrData
827.00 ms   21.6%   0 s                    Note.init(url:)
823.00 ms   21.5%   1.00 ms                 specialized Note.init(url:)
521.00 ms   13.6%   1.00 ms                  @objc CoreNote.init(fileURL:)
519.00 ms   13.6%   0 s                       CoreNote.init(fileURL:)
514.00 ms   13.4%   3.00 ms                    -[UIDocument initWithFileURL:]
```
Maybe running the System Usage report in Instruments would help, but it doesn't work in the Simulator and I couldn't get a development build of the app onto my phone.

**Comment by glushchenko** [comment]
Thank you for feedback. Fast loading in my priorirties. I think it will be fixed in next release.

**Comment by glushchenko** [comment]
Background loading added in 1.8

**Comment by afaden** [comment]
Thanks very much! I see the new DispatchQueue code [here](https://github.com/glushchenko/fsnotes/commit/52f91ad20e93c2063f8d42811172b87cf77114f3#diff-3ecfa2af0344868a70aef8f91dca2311). I didn't realize when I was playing around with the code that you could call storage.initiateCloudDriveSync and storage.loadDocuments in parallel, even better! I'll keep an eye out for the 1.8 update and let you know how it works with my folder of notes.

**Comment by afaden** [comment]
Using 1.8 now, it's no longer crashing on startup, thanks! That said, it takes about ~45 seconds to load the list of notes, but I'll open a separate issue for that.

**Comment by glushchenko** [comment]
@afaden sorry, not fixed yet. I think after UI update i will do it.

**Comment by afaden** [comment]
I’d be happy to help test the betas, but I don’t currently have access.

**Comment by glushchenko** [comment]
Please mail me
